build/common.sh

```diff
@@ -55,7 +43,16 @@ readonly KUBE_BUILD_PPC64LE="${KUBE_BUILD_PPC64LE:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat ${KUBE_ROOT}/build/build-image/cross/VERSION)"
-# KUBE_BUILD_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+# KUBE_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+
+# This version number is used to cause everyone to rebuild their data containers
+# and build image.  This is especially useful for automated build systems like
```

Comment:
Hmm -- Tim had some issues with this at one point where he was mucking this stuff out.  But we should still do this as it'll trip up others.